pfcpiface/p4rtc.go

```diff
@@ -49,6 +52,50 @@ type P4rtClient struct {
 	P4Info p4ConfigV1.P4Info
 }
 
+type P4RuntimeError struct {
+	errors []*p4.Error
+}
+
+func (e *P4RuntimeError) Error() string {
+	return fmt.Sprintf("P4RuntimeError: %v", e.errors)
+}
+
+func (e *P4RuntimeError) Get() []*p4.Error {
+	return e.errors
+}
+
+// convertError parses nested P4Runtime errors.
+// See https://p4.org/p4-spec/p4runtime/main/P4Runtime-Spec.html#sec-error-reporting-messages.
+func convertError(err error) error {
```

Comment:
This addition seems like a standalone PR. It's self-contained to `pfcpiface/p4rtc.go`.

Reply:
Right. See https://github.com/omec-project/upf-epc/pull/429.